src/Okta.Sdk/OktaClient.cs

```diff
@@ -69,6 +69,8 @@ public OktaClient(OktaClientConfiguration apiClientConfiguration = null, ILogger
                 serializer,
                 resourceFactory,
                 logger);
+
+            PayloadHandler.TryRegister<PkixCertPayloadHandler>();
```

Comment:
Is Json payload already registered?

Reply:
Yes, Json payload is the default and is always registered.